packages/dashboard/src/components/ConversationTable.tsx

```
@@ -1,92 +1,142 @@
+            {displayConversations.map((conversation) => (
+              <tr 
+                key={conversation.promptId} 
+                className={onConversationClick ? 'cursor-pointer' : ''}
+                onClick={() => onConversationClick?.(conversation)}
+              >
+                <td>
+                  <div className="d-flex flex-column">
+                    <div className="fw-medium">
+                      {filterState.search ? (
+                        <span dangerouslySetInnerHTML={{
+                          __html: highlightSearchTerm(conversation.promptId, filterState.search)
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)